cluster/options.go

```diff
@@ -4,8 +4,16 @@ import "crypto/tls"
 
 // Options is exported
 type Options struct {
```

Comment:
Yes that was what I was thinking too, we need to be able to use certs separately (even though I don't know what it's going to look like in terms of UI with the `cli`)